src/gql/query/users.ts

```diff
@@ -30,6 +47,13 @@ export const usersQueryString = (
   )
 }
 
+export const user = async (
```

Comment:
Would argue that this should live in it's own file called user.ts

Reply:
Moved to a new file